### DISCUSSION TITLE
adding chord progressions

### DISCUSSION THREAD

**Discussion by darrenburgess** [ShacharHarshuv/open-ear#260] [General]
I have been working with Open-Ear quite a bit.  I would like to add chord progressions to that exercise.  Is it just a matter of editing commonProgressions.ts and submitting a pull request? @ShacharHarshuv

**Comment by ShacharHarshuv** [comment]
Hey, 
Yes - `commonProgression.ts` is for the "Common Progressions" exercise. Editing the `commonProgressionDescriptorList` value should add things to that list. 
If you want the "Chord Progressions In Real Songs" exercise (unfortunately only available on android at this moment) you will need to edit `chordsInRealSongsDescriptorList.ts` instead.

**Reply by darrenburgess** [reply]
ok.  I guess I will have to get me an android device.  I was wondering why I did not see that exercise.

I see there was a comment about this in another discussion from a awhile ago.  You raised a concern about the UI implications about adding allot of items to a list I think.  

that said, the app is useful with deeper exercises even if the UI is a bit wonky.  I will add to the chord progressions soon.

**Comment by ShacharHarshuv** [comment]
> ok. I guess I will have to get me an android device. I was wondering why I did not see that exercise.

Or try to get it working on iOS. I had some difficulties but I think a lot of people will be happy if you can figure it out. 

> I see there was a comment about this in another discussion from a awhile ago. You raised a concern about the UI implications about adding allot of items to a list I think.

Yes, we might want to change the component to have search / something. But I think you can still add progressions if you want.
